### ISSUE TITLE
Yearly repeating events have stacking number at the end

### ISSUE THREAD

**Issue by Phl3gmaTREEc** [SimpleMobileTools/Simple-Calendar#1672]
Creating yearly repeating events in my Google calendar puts stacking number at the end.
For example event created for today would be called "test event" but next year it would be "test event (1)" year after that "test event (2).
This is also the case for events created way in the past.
This shows only in this application not in native Google calendar.

**Comment by tibbi** [comment]
Arent they birthdays or anniversaries created with the top menu buttons? I dont think it is shown as manually created events.

**Comment by Phl3gmaTREEc** [comment]
No, they are normally created events. It shows even on events created way before I've ever used this application.

**Comment by tibbi** [comment]
If you check the Edit event screen, do you still see the numbers there?

**Comment by Phl3gmaTREEc** [comment]
No, in edit mode no numbers.

**Comment by tibbi** [comment]
I dont thats possible. Please make a screen recording with creating a yearly event and seeing the number in brackets

**Comment by Phl3gmaTREEc** [comment]
How can I share that video with you?

**Comment by tibbi** [comment]
upload it here on github

**Comment by Phl3gmaTREEc** [comment]
https://user-images.githubusercontent.com/77894945/158558008-9f6b36a7-870e-420d-a2c8-0e2877f4612d.mp4
There you go

**Comment by tibbi** [comment]
right, so it happens at Anniversaries and Birthdays event type/calendar only, no? I mean the bottom field.

**Comment by Phl3gmaTREEc** [comment]
Ok, this is bizarre. The "anniversaries" thing is a another calendar inside my main google calendar created to separate this type of event. Now I created another calendar called "test calendar" and it works normally.
My best guess is that google recognized the name Anniversaries and all events in that calendar were automatically counted for how many years ti has been since first event.
You have to love automation, that you don't know is there.

Sorry about this.

**Comment by tibbi** [comment]
the number is added by us, but it is supposed to be added only at local events created by the top Menu -> Add contact birthdays/anniversaries, not at CalDAV synced ones. Will take a look at it. Thanks

**Comment by tibbi** [comment]
should be fixed in 6.17.4, unless it wont be
